.github/config/release-config.json

```diff
@@ -111,6 +111,22 @@
         ],
         "exhaustive": true,
         "exhaustive_rules": false
+      },
+      {
```

Comment:
@cosimomeli this configuration update needs to be applied to relevant repositories